src/Iris/Colour/Formatting.hs

```diff
@@ -20,16 +20,16 @@ module Iris.Colour.Formatting
 
 import Control.Monad.IO.Class (MonadIO (..))
 import Control.Monad.Reader (MonadReader)
-import Data.ByteString (ByteString)
+import qualified Data.Text as T
```

Comment:
GHC outputs the warning that this import is redundant. I believe, only `IO` functions from `Text` are used so this one can be removed 🧹

Reply:
Removed the redundant import, though it now won't compile though as the simple-grep uses ByteString still.